src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -461,6 +463,33 @@ private String removeBrackets(String tag) {
     return tag.replaceAll("[\\[\\]]", "");
   }
 
+  /**
+   * Called from JavaScript when user hovers over a country flag.
+   */
+  public void countryInfo(String flagPath) {
+    countryInfoPopup = new Popup();
```

Comment:
Some people dont have a flag or a ? flag ...What happens then?

Reply:
The onmouseover JavaScript event is bound to the image.
No image -> nothing to hover over -> no event.

Reply:
See chat_section.html.